src/inmanta/agent/agent.py

```diff
@@ -372,40 +372,58 @@ class DeployRequestAction(str, enum.Enum):
     """ interrupt the current deploy: cancel it, start the new one and restart the current deploy when the new one is done"""
 
 
+# Two letter abbreviations to make table line up
+# N normal
+# P periodic
+# F full
+# I incremental
+NF = (True, False)
+NI = (False, False)
+PF = (True, True)
+PI = (False, True)
```

Comment:
Slight preference to swap the letters so their order matches the order in the tuple.